recipe/recipe.yaml

```diff
@@ -0,0 +1,82 @@
+context:
+  name: pyright
+  version: 1.1.386
+
+recipe:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+build:
+  number: 10
+
+source:
+  url: https://pypi.io/packages/source/${{ name[0] }}/${{ name }}/pyright-${{ version }}.tar.gz
+  sha256: 8e9975e34948ba5f8e07792a9c9d2bdceb2c6c0b61742b068d2229ca2bc4a9d9
+
+outputs:
+  - package:
+      name: ${{ name|lower }}-node
+    build:
+      skip: [ppc64le]
+      script:
+        file: build_node${{ '.bat' if win else '.sh' }}
+    requirements:
+      build:
+        - pnpm
+        - pnpm-licenses
+    about:
+      license: MIT
+      license_file: third-party-licenses.txt
+
+  - package:
+      name: ${{ name|lower }}
+    build:
+      noarch: python
+      script:
+        file: build_python${{ '.bat' if win else '.sh' }}
+      python:
+        entry_points:
+          - pyright=pyright.cli:entrypoint
+          - pyright-python=pyright.cli:entrypoint
+          - pyright-langserver=pyright.langserver:entrypoint
+          - pyright-python-langserver=pyright.langserver:entrypoint
+
+    requirements:
+      host:
+      - python
+      - setuptools
+      - pip
+      run:
+      - python
+      - nodejs
+      - nodeenv >=1.6.0
+      - typing-extensions >=4.1
+      - ${{ pin_subpackage(name ~ '-node') }}
```

Comment:
![image](https://github.com/user-attachments/assets/a158893b-cd4e-41bc-bd29-4b90bc0aa67a)
this doesn't look ideal. 

```suggestion
      - ${{ pin_subpackage(name ~ '-node', max_pin='x.x.x') }}
```

should probably do the trick